### ISSUE TITLE
Fields in Orchard display as "&amp;" instead of "&"

### ISSUE THREAD

**Issue by LegendORC** [OrchardCMS/Orchard#7111]
I just download Orchard 1.10.
and create new Field on Blog as `Start Date & Time`.
When i try to create new Blog it's display `Start Date & Time` correctly on the UI.
But... when i click on the "Save" button without put any data.
Error message will display as `Start Date &amp; Time is required.` instead of `Start Date & Time is required.`

![orchardbug](https://cloud.githubusercontent.com/assets/5640085/18375747/1070c6e0-7683-11e6-96e0-ea161d66d34c.png)

Thank you.

**Comment by sebastienros** [comment]
Because AddModelError will re-encode the error message, we need to prevent `T` from encoding the name by wrapping it in an `IHtmlString`
https://github.com/OrchardCMS/Orchard/blob/dev/src/Orchard.Web/Modules/Orchard.Fields/Drivers/DateTimeFieldDriver.cs#L158

**Comment by rtpHarry** [comment]
@sebastienros I have looked into this issue. I searched the solution with this regex to see if others were affected by it:

    updater\.AddModelError\(.*DisplayName.*\);

Some of them used a slightly different pattern:

    updater.AddModelError(GetPrefix(field, part), T("{0} is required.", T(field.DisplayName)));

See the param is wrapped in a `T()` as well. This fixes the issue:

![image](https://cloud.githubusercontent.com/assets/1038062/18609378/bf3ae996-7cf8-11e6-97b6-95009613ca1e.png)

Is this the solution or a workaround?
